davealex/quick-credit
# #165588956 Design Admin Post Loan Repayment Page

#### What does this PR do?
This PR adds an admin_post_loan_repayment.html page to the Project
#### Description of Task to be completed?
On this page, an admin can see a simple form that accepts a reference each to a loan and the client, and the value of the repayment sum to be posted actions on an application; reject or approve
#### How should this be manually tested?
Navigate to the admin_post_loan_repayment.html page and ensure that the page is properly rendered when served on mobile devices
Also, ensure that the client-side form validation works as expected
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#165588956
#### Screenshots (if appropriate)
#### Questions:
